### ISSUE TITLE
LogAnalyzer can not handle large files

### ISSUE THREAD

**Issue by renzon** [SatelliteQE/automation-tools#632]
Check https://github.com/SatelliteQE/satellite6-upgrade/issues/83

**Comment by san7ket** [comment]
There was a failure recently on the job due loganalyzer. when there was a non utf8 character in the log file

```
Traceback (most recent call last):
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/main.py", line 762, in main
    *args, **kwargs
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 425, in execute
    results['<local-only>'] = task.run(*args, **new_kwargs)
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 172, in run
    return self.wrapped(*args, **kwargs)
  File "/home/jenkins/workspace/upgrade-to-6.3-rhel7/upgrade/runner.py", line 233, in product_upgrade
    host=cap_host)
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/automation_tools/satellite6/log.py", line 80, in __exit__
    execute(fetch_appended_log_lines, host=self.host)
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 385, in execute
    multiprocessing
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 275, in _execute
    return task.run(*args, **kwargs)
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 172, in run
    return self.wrapped(*args, **kwargs)
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/automation_tools/satellite6/log.py", line 78, in fetch_appended_log_lines
    analyze(self.host, log_file, content)
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib/python2.7/site-packages/automation_tools/satellite6/log.py", line 146, in analyze
    content_lines = content.decode('utf-8').split('\n')
  File "/home/jenkins/shiningpanda/jobs/47b1311d/virtualenvs/d41d8cd9/lib64/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe0 in position 813131: invalid continuation byte
```

Should I create a different issue for this ?

**Comment by renzon** [comment]
No need, I am going to fix it on #636
